quickstart/sparcv8leon3-gr716-mini.md

```diff
@@ -0,0 +1,145 @@
+
+First, check on which port the board is connected to the computer. To do this, run the following command:
+
+```bash
+ls -l /dev/serial/by-id
+```
+
+<img src="_images/gr716-ls.png" width="700px">
+
+In this case, the debug UART is connected to the `ttyUSB0` port.
+Launch the `GRMON` monitor using the following command:
+
+```bash
+grmon -uart /dev/ttyUSB0 -baud 115200
+```
+
+<details>
+<summary>How to get GRMON</summary>
+
+- Download the GRMON software from the [official website](https://www.gaisler.com/index.php/downloads/debug-tools).
+- After downloading the archive, extract it and optionally add the `grmon` binary to the `PATH` variable.
+
+</details>
+</br>
+
+The `-baud` parameter specifies the baud rate of the `AHBUART1` interface.
+Optionally you can pass  the `-gdb` parameter, which enables the GDB server on port 2222.
+Default CPU clock frequency is 50 MHz.
+
+To load the bootloader (`plo`) to the RAM, run the following commands in the `GRMON` monitor:
+
+```bash
+load phoenix-rtos-project/_boot/sparcv8leon3-gr716-mini/plo-ram.img 0x31000000
+```
+
+To verify that the file has been loaded correctly, run the following command:
+
+```bash
+verify phoenix-rtos-project/_boot/sparcv8leon3-gr716-mini/plo-ram.img 0x31000000
+```
+
+Set the entry point of the bootloader:
+
+```bash
+ep 0x31000000
+```
+
+Open a new terminal window and run the following command:
+
+```bash
+picocom -b 115200 --imap lfcrlf /dev/ttyUSB2
+```
+
+This will connect to the `UART2` interface, which is used for the console.
+To start the bootloader, execute the following command in the `GRMON` monitor:
+
+```bash
+go
+```
+
+The bootloader interface should appear in the console.
+
+<img src="_images/gr716-plo.png" width="700px">
+
+### Copying flash image using PHFS (phoenixd)
+
+To flash the disk image, first, you need to verify on which port plo USB device has been appeared.
+You can check that using `ls` as follow:
+
+```bash
+ls -l /dev/serial/by-id
+```
+
+To share disk image to the bootloader, `phoenixd` has to be launched with the following arguments
+(choose suitable ttyUSBx device, in this case, `ttyUSB3`):
+
+```bash
+sudo ./phoenixd -p /dev/ttyUSB3 -b 115200 -s .
+```
+
+<img src="_images/gr716-phoenixd.png" width="700px">
+
+To start copying a file, write the following command in the console with plo interface:
+
+```bash
+copy uart3 phoenix.disk flash0 0x0 0x0
+```
+
+<img src="_images/gr716-copy.png" width="700px">
+
+The `flash0` is the external flash memory.
+
+After copying is done, reset the board and the system should start. To reboot, write `reboot` command in the PLO
+console.
+
+## Using Phoenix-RTOS
+
+After reboot, Phoenix-RTOS will be launched and the `psh` shell command prompt will appear in the terminal with
+the serial port opened.
+
+<img src="_images/gr716-start.png" width="700px">
+
+- Note: You can also enter plo (Phoenix-RTOS loader) by pressing any button, for example,
+`enter` within some time after reset.
```

Comment:
This clarification is better suited for the start of the document. Maybe a comment from where this time comes from?
```suggestion
- Note: You can also enter `plo` by pressing any button within some time after reset.
```